tcn-client-android/src/main/java/org/tcncoalition/tcnclient/TcnConstants.kt

```diff
@@ -35,4 +35,7 @@ object TcnConstants {
 
     private const val H_TCK_DOMAIN_SEPARATOR_STRING = "H_TCK"
     val H_TCK_DOMAIN_SEPARATOR = H_TCK_DOMAIN_SEPARATOR_STRING.toByteArray(Charsets.UTF_8)
+
+    const val WAKELOCK_TAG = "org:tcn-coalition:alarm-receiver"
+    const val WAKELOCK_DURATION = 10000 // This should be set to the longest amount of time it would take to do a scan
```

Comment:
Any particular reason to have chosen this duration, or is it more a placeholder?

Reply:
Hey sorry for the delayed response.  That's the maximum time.  You can set the lock for less, but then you might get killed before you are done scanning.  The way that you have things implemented (using a connection, which you need to because of iOS) you want all the time you can get.  If there are enough handsets in the room 10 seconds might not be enough time.